### PR TITLE
Soupcan migration

### LINKED ISSUE #300: extension integration (soupcan) not working
I have no idea if its a blueblocker issue or a soupcan issue but the soupcan integration hasn't been working since the 0.4.2 update and i cant seem to find the problem anywhere

### PR DESCRIPTION
**Pull request by rougetimelord**
Migrates Soupcan to the new integration system. Closes #300.

I don't use Soupcan so I can't test this, I think it's right but someone should test it :)

### CODE REVIEW

File: src/shared.ts [hunk 1]
```diff
@@ -735,35 +760,25 @@ export async function BlockBlueVerified(user: BlueBlockerUser, config: CompiledC
...
+
+				integrations[SoupcanExtensionId] = {
+					name: 'Soupcan',
+					state: IntegrationStateSendAndReceive,
+				};
+
+				config.soupcanIntegration = false;
+				api.storage.local.set({ config });
+			}
+
```
Comment (rougetimelord) [inline src/shared.ts:781]: This is the important part the rest is formatting, my bad

**Comment by cooljeanius** [comment]
I don't feel qualified to review this one, but maybe @fred-ronnoc wants to take a look?

**Comment by rougetimelord** [comment]
Going to ping @bethylamine here too, hi hi!

**Comment by rougetimelord** [comment]
Tested this, migration works correctly

**Comment by rougetimelord** [comment]
@cooljeanius, I think we're good to merge here

**Review by cooljeanius — APPROVED** [review]
OK, I guess

**Pull request merged**